internal/docs/design.md

```diff
@@ -115,12 +115,12 @@ it is needed.
 
 The exception we permit is URL scheme registration as documented under
 [URLs](#urls). The amount of boilerplate setup code required for URL muxes for
-multiple providers without use of a tool like Wire is an unreasonable burden for
+multiple drovers without use of a tool like Wire is an unreasonable burden for
```

Comment:
Is this a type or a compromise solution drover = providers + driver? :)

Reply:
Ha - no. Fixed to `driver`.